STTBot/events/app_mention.py

```diff
@@ -394,6 +402,16 @@ def _ret_success_blocks(blocks, suc_message, say):
         "help": "Adds reactions for pin showdown",
         "func": _cmd_poll_react
     },
+    {
+        "cmd": "STT",
+        "sub_cmd": "Draft",
+        "args": [
+            "list"
+        ],
+        "help": "Randomises order of users for STT Draft",
+        "func": _cmd_STT_draft
```

Comment:
This needs to match the function name on 305

Reply:
done, silly error